presto-parquet/src/main/java/com/facebook/presto/parquet/ParquetCorruptStatisticsUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.parquet;
+
+import com.google.common.base.Strings;
+import org.apache.parquet.SemanticVersion;
+import org.apache.parquet.VersionParser;
+import org.apache.parquet.hadoop.metadata.FileMetaData;
+
+import java.util.Optional;
+
+public class ParquetCorruptStatisticsUtils
+{
+    private static final SemanticVersion PARQUET_1025_CHANGED_VERSION = new SemanticVersion(1, 10, 0);
```

Comment:
could you please add comments here? explain why before Parquet 1025 might have corrupted statistics
s/PARQUET_1025_CHANGED_VERSION/PARQUET_CORRUPT_STATISTICS_VERSION/g